File: camunda-carbon-reductor-c7/src/main/java/de/envite/greenbpm/carbonreductorconnector/CamundaConnectorApplication.java
package de.envite.greenbpm.carbonreductorconnector;

import de.envite.greenbpm.carbonreductor.core.EnableCarbonReductorCore;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@EnableCarbonReductorCore
public class CamundaConnectorApplication {

    public static void main(String[] args) {
        SpringApplication.run(CamundaConnectorApplication.class, args);
    }
}

File: camunda-carbon-reductor-c7/src/main/kotlin/de/envite/greenbpm/carbonreductorconnector/CamundaConnectorApplication.kt
package de.envite.greenbpm.carbonreductorconnector

import de.envite.greenbpm.carbonreductor.core.EnableCarbonReductorCore
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableCarbonReductorCore
class CamundaConnectorApplication

fun main(args: Array<String>) {
    runApplication<CamundaConnectorApplication>(*args)
}